tools/compiler/src/parser.rs

```diff
@@ -74,5 +74,6 @@ pub fn get_chain_configs() -> AppResult<Vec<ChainConfig>> {
             };
             Ok(chain_config)
         })
-        .collect()
+        .filter_map(|result| result.ok())
```

Comment:
Don't we want to throw if the user did not set up the file structure correctly? Wouldn't this mean it kinda would silently fail?

Reply:
Yeah, I did this along the way to starting towards having it consume a git submodule and change the input directory structure, which I abandoned and then wrote up as a follow on issue. Getting output of some kind felt (feels) more important